lib/opentracing/tracer.rb

```diff
@@ -24,12 +24,10 @@ def inject(span_context, format, carrier)
     end
 
     # Extract a span from a carrier
```

Comment:
And the `inject` signature is wrong, too. I think I'll just make a PR to fix this properly.

Sorry again for not-noticing this. Please feel free to @bensigelman me if I'm not paying attention to something.